files/quickjs/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "quickjs",
+  "repo": "mk2018/quickjs",
+  "files": {
+    "basePath": "builds/0.1a/",
```

Comment:
This format wont work, it will break the minute you release a new version.
Also there is no documentation or website.